src/pipecat/services/elevenlabs.py

```diff
@@ -167,34 +180,6 @@ async def set_model(self, model: str):
         await self._disconnect()
         await self._connect()
 
-    async def set_voice(self, voice: str):
-        logger.debug(f"Switching TTS voice to: [{voice}]")
-        self._voice_id = voice
-        await self._disconnect()
-        await self._connect()
-
-    async def set_voice_settings(
-        self,
-        stability: Optional[float] = None,
-        similarity_boost: Optional[float] = None,
-        style: Optional[float] = None,
-        use_speaker_boost: Optional[bool] = None,
-    ):
-        self._params.stability = stability if stability is not None else self._params.stability
-        self._params.similarity_boost = (
-            similarity_boost if similarity_boost is not None else self._params.similarity_boost
-        )
-        self._params.style = style if style is not None else self._params.style
-        self._params.use_speaker_boost = (
-            use_speaker_boost if use_speaker_boost is not None else self._params.use_speaker_boost
-        )
-
-        self._set_voice_settings()
-
-        if self._websocket:
-            msg = {"voice_settings": self._voice_settings}
-            await self._websocket.send(json.dumps(msg))
```

Comment:
to expand on above... maybe this needs to override `_update_settings()` to be something like:

```
async def _update_settings(self, settings: Dict[str, Any]):
  super()._update_settings(settings)
  self._voice_settings = self._set_voice_settings()
  if self._websocket:
    msg = {"voice_settings": self._voice_settings}	
    await self._websocket.send(json.dumps(msg))
```

Reply:
apparently this function was not needed to begin with and should be removed and functionality not preserved, so this can be ignored... but not the one above about `set_voice`